packages/hooks/src/useSendOTPVerificationCode.tsx

```diff
@@ -69,6 +71,12 @@ const useSendOTPVerificationCode = () => {
     // Usage in useEffect
     useEffect(() => {
         if (phone_otp_error) {
+            trackPhoneVerificationEvents({
+                action: 'error',
+                subform_name: 'verify_phone_otp_screen',
+                // @ts-expect-error will remove once solved
+                error_code: phone_otp_error.code,
+            });
```

Comment:
are we able to send from which screen session timeout happended?

Reply:
yes we could do that but `subform_name` is being replaced by 'session_timeout_popup' already, I'll ask irene whether we can move 'session_timeout_popup' to action and `subform_name` we put based on which screen we at

Reply:
@amina-deriv I've checked with Irene she said we can do that, need to check with Maria about the new action for this 🙏